third_party/nvfuser/python/__init__.py

```diff
@@ -1 +1,17 @@
 from ._C import *
+
+class FusionDefinition(_C._FusionDefinition):
+    def from_pytorch(self, tensor) :
+        """
+        Defines an nvfuser input tensor from a pytorch tensor
+        """
+        try:
+            from .pytorch_utils import torch_dtype_to_nvfuser_dtype
+        except ImportError:
+            raise ImportError("Unable to import pytorch_utils!")
+
+        if not tensor.is_cuda:
+            raise ValueError("Tensor should be on a cuda device!")
+
+        return self.define_tensor(sizes=tensor.size(), strides=tensor.stride(),
```

Comment:
I think this is fine for now. I'm not a big fan of silently converting it to static size. I'm pretty sure user will just follow our example and come back complaining about static shapes later.

Reply:
This API does not create static sizes.  It converts to symbolic sizes and determines contiguity underneath.

Reply:
ah, sorry I think we had this conversation before.. I keep forgetting it..